crc-org/crc
# Add 'watch' argument to status command

**Fixes:** Issue #N

**Relates to:** #2876

## Solution/Idea

This will display CPU and RAM load in real time.
To achieve this PR add websocket implementation and daemon send data through websocket `/status` endpoint.

It is use https://github.com/nhooyr/websocket as websocket implementation
and upgrade https://github.com/cheggaaa/pb to `v3` to render load bars in terminal.

Demo:

https://user-images.githubusercontent.com/929743/208875792-e665602a-548d-4f71-bda9-fd435bfc70ac.mov


## Testing

1. run any preset
2. call `crc status -w`
3.  look on output, it should contains bar for RAM and bars for CPU's load
